backend/kesaseteli/kesaseteli/settings.py

```diff
@@ -196,6 +203,26 @@
 EAUTHORIZATIONS_CLIENT_SECRET = env.str("EAUTHORIZATIONS_CLIENT_SECRET")
 EAUTHORIZATIONS_API_OAUTH_SECRET = env.str("EAUTHORIZATIONS_API_OAUTH_SECRET")
 
+# Azure ADFS
+LOGIN_URL = "django_auth_adfs:login"
+
+ADFS_CLIENT_ID = env.str("ADFS_CLIENT_ID")
+ADFS_CLIENT_SECRET = env.str("ADFS_CLIENT_SECRET")
+ADFS_TENANT_ID = env.str("ADFS_TENANT_ID")
+
+AUTH_ADFS = {
+    "AUDIENCE": ADFS_CLIENT_ID,
```

Comment:
for us, is AUDIENCE the same as ADFS_CLIENT_ID in all environments?

Reply:
Yes. More info in https://django-auth-adfs.readthedocs.io/en/latest/azure_ad_config_guide.html#step-2-configuring-settings-py

Reply:
ok